src/style.css

```diff
@@ -1,3 +1,74 @@
+:root {
+  --text-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  height: 60px;
```

Comment:
Need to fix it
<img width="377" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/43563a16-1ae4-4c4b-ae77-d1cfc4390eeb">
```suggestion
```